### PR TITLE
added different particle types exclusive interactions to md

### PR DESCRIPTION
**Pull request by maximillian-dolan**
We aspire to deal with all pull requests in a timely manner. Please be patient.

Ideally pull requests will respect the underlying intrastructure of pylj and agree with PEP8, etc. However, please do not let this put novice programmers off contributing.

### CODE REVIEW

File: examples/simple_examples/molecular_dynamics.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
    "outputs": [],
    "source": [
     "import warnings\n",
-    "from pylj import md, sample\n",
+    "from pylj import md, sample, pairwise\n",
```
Comment (arm61) [inline examples/simple_examples/molecular_dynamics.ipynb:10]: Why are you importing `pairwise` here?

File: pylj/pairwise.py [hunk 2]
```diff
@@ -95,11 +110,12 @@ def update_accelerations(particles, f, m, dx, dy, dr):
     k = 0
     for i in range(0, particles.size - 1):
         for j in range(i + 1, particles.size):
-            particles["xacceleration"][i] += second_law(f[k], m, dx[k], dr[k])
-            particles["yacceleration"][i] += second_law(f[k], m, dy[k], dr[k])
-            particles["xacceleration"][j] -= second_law(f[k], m, dx[k], dr[k])
-            particles["yacceleration"][j] -= second_law(f[k], m, dy[k], dr[k])
+            particles["xacceleration"][i] += second_law(f[k], m, dx[k], dr[k]) if f[k]!=0 else 0
```
Comment (arm61) [inline pylj/pairwise.py:113]: Can you put this through a linter? It looks ugly just now...

File: pylj/pairwise.py [hunk 3]
```diff
@@ -171,54 +187,15 @@ def lennard_jones_force(A, B, dr):
     float:
         The force between the two particles.
     """
-    print(
+    xacceleration(
```
Comment (arm61) [inline pylj/pairwise.py:190]: `xacceleration`?

**Review by arm61 — COMMENTED** [review]
Some minor comments.

File: examples/simple_examples/molecular_dynamics.ipynb (hunk 1 above)
Comment (maximillian-dolan) [reply examples/simple_examples/molecular_dynamics.ipynb:10]: left in accidentally after tests, now removed

File: pylj/pairwise.py (hunk 3 above)
Comment (maximillian-dolan) [reply pylj/pairwise.py:190]: mistakenly changed, put back to print now

**Pull request merged**